content/en/docs/languages/_includes/index-intro.md

```diff
@@ -0,0 +1,24 @@
+---
+---
+
+This is the OpenTelemetry {{ $name }} documentation. OpenTelemetry is an
```

Comment:
I know this introduction is how it's been done for ages, but I find "This is" a bit weak, like it points out the obvious. Maybe we could say something like "Welcome to the OpenTelemetry (name) documentation...."?

Reply:
Agreed. Let's tackle that change in a followup PR.